src/tools/perf/lib/ucp_tests.cc

```diff
@@ -659,14 +664,26 @@ class ucp_perf_test_runner {
 
         ucx_perf_omp_barrier(&m_perf);
 
-        if (my_index == 0) {
+        if (m_perf.params.flags & UCX_PERF_TEST_FLAG_LOOPBACK) {
+            UCX_PERF_TEST_FOREACH(&m_perf) {
+                send(ep, send_buffer, send_length, send_datatype,
```

Comment:
See `clang-format` output.
Generally, unless there is a specific reason not to do so, we usually apply its suggestions.